google/cloud/forseti/services/server.py

```diff
@@ -170,6 +196,10 @@ def main():
         '--enable_console_log',
         action='store_true',
         help='Print log to console.')
+    parser.add_argument(
+        '--enable-tracing',
+        action='store_true',
```

Comment:
Is this, and should be, enabled?  Per your PR description, this should be disabled by default?

Reply:
Yes, `store_true` actually stores False by default in Python (cf: https://docs.python.org/2/library/argparse.html#action)

Reply:
TIL  :)